src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -11459,11 +11459,11 @@ public enum E { }
                 }
 
                 [A(default(B<delegate*<void>[]>.E))]
```

Comment:
>[A(default(B<delegate*<void>[]>.E))] [](http://example.com/codeflow?start=16&length=36)

Is there a breaking change around this scenario in C#12?

Reply:
I don't think there's a breaking change, but the error changed. Will add both tests side by side.